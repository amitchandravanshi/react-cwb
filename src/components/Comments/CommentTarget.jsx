import React from "react";
import { withStyles } from "@material-ui/core/styles";

const styles = theme => {
  return {
    root: {
      width: "85%",
      paddingLeft: "1rem",
      fontFamily: "AvenirNextLT-Regular,AvenirNextLT-Demi"
    }
  };
};

const CommentTarget = props => {
  const { classes, targetText } = props;
  return <pre className={classes.root}>{targetText}</pre>;
};

export default withStyles(styles)(CommentTarget);
