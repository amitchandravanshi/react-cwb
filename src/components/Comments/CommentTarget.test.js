import React from "react";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
import CommentTarget from "./CommentTarget";
const props = {
  classes: {}
};
describe("<CommentTarget> ", () => {
  it("<CommentControls> snapshot ", () => {
    const wrapper = mount(<CommentTarget {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
