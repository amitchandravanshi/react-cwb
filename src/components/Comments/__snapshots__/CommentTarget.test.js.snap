// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommentTarget>  <CommentControls> snapshot  1`] = `
<CommentTarget
  classes={
    Object {
      "root": "CommentTarget-root-1",
    }
  }
/>
`;
